pyproject.toml

```diff
@@ -8,6 +8,7 @@ authors = ["Ben Beecher <Ben@lightmatter.com>"]
 python = "^3.9"
 cookiecutter = "^1.7.3"
 isort = "^5.10.1"
+heroicons = "^1.8.0"
```

Comment:
I think you added this to the wrong pyproject. You're looking for the one inside the cookiecutter

Reply:
Nice catch! Probably did this incorrectly with Poetry.